src/base/compute/owl_computation_symbol.ml

```diff
@@ -376,32 +465,30 @@ module Make
   let is_const x = (attr x).op = Const
 
 
-  let is_arr x =
+  let is_node_arr x =
```

Comment:
let's leave a mark here to remind us to clean up the interface later. e.g., using `op` type for a more reliable version of `is_elt` and `is_arr`